mne/source_space.py

```diff
@@ -229,50 +229,35 @@ def export_volume(self, fname, include_surfaces=True,
         # Get shape, inuse array and interpolation matrix from volume sources
         first_vol = True  # mark the first volume source
         # Loop through the volume sources
-        for vs in src_types['volume']:
+        inuse = 0
+        for ii, vs in enumerate(src_types['volume']):
             # read the lookup table value for segmented volume
             if 'seg_name' not in vs:
                 raise ValueError('Volume sources should be segments, '
                                  'not the entire volume.')
             # find the color value for this volume
-            i = _get_lut_id(lut, vs['seg_name'], use_lut)
+            id_ = _get_lut_id(lut, vs['seg_name'], use_lut)
 
-            if first_vol:
+            if ii == 0:
                 # get the inuse array
                 if mri_resolution:
                     # read the mri file used to generate volumes
-                    aseg = nib.load(vs['mri_file'])
-
+                    aseg_data = nib.load(vs['mri_file']).get_data()
                     # get the voxel space shape
                     shape3d = (vs['mri_height'], vs['mri_depth'],
                                vs['mri_width'])
-
-                    # get the values for this volume
-                    inuse = i * (aseg.get_data() == i).astype(int)
-                    # store as 1D array
-                    inuse = inuse.ravel((2, 1, 0))
-
                 else:
-                    inuse = i * vs['inuse']
-
                     # get the volume source space shape
-                    shape = vs['shape']
-
                     # read the shape in reverse order
                     # (otherwise results are scrambled)
-                    shape3d = (shape[2], shape[1], shape[0])
-
+                    shape3d = vs['shape'][2::-1]
                 first_vol = False
-
+            if mri_resolution:
+                # get the values for this volume
+                use = id_ * (aseg_data == id_).astype(int).ravel('F')
             else:
-                # update the inuse array
-                if mri_resolution:
-
-                    # get the values for this volume
-                    use = i * (aseg.get_data() == i).astype(int)
-                    inuse += use.ravel((2, 1, 0))
```

Comment:
This is a fix for deprecated use of indices in `ravel`, can split into a different PR if necessary, but both are needed for 0.11